torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3192,10 +3192,6 @@ def sample_inputs_conv1d(op_info, device, dtype, requires_grad, **kwargs):
         ((1, 4, 5), (3, 4, 3), None, {}),
     )
 
-    # TODO: (@krshrimali), add error_inputs_func once https://github.com/pytorch/pytorch/pull/67354 is merged
-    # Should replace test_conv_modules_raise_error_on_incorrect_input_size and test_conv_shapecheck
-    # in test/test_nn.py
```

Comment:
Do we cover the `test_conv_modules_raise_error_on_incorrect_input_size` and `test_conv_shapecheck` from test/test_nn.py? If so, we should remove that test from `test_nn.py`

Reply:
Yeah.
- `test_conv_shapecheck` includes shape checks relating to groups, strides, padding, dilation, and incorrect shapes which I included.
- `test_conv_modules_raise_error_on_incorrect_input_size` includes checks relating to invalid dims, I added these too.